draft-working-group-charter/index.html

```diff
@@ -0,0 +1,605 @@
+                        </p>
+                    </dd>
+                </dl>
+            </div>
+
+            <div id="timeline">
+                <h3>Timeline (TBD)</h3>
+
+                <table class="roadmap">
+                    <tfoot>
+                        <tr>
+                            <td colspan="5" rowspan="1">
+                                Note: The group will document significant changes from this initial schedule on the group home page.
+                            </td>
+                        </tr>
+                    </tfoot>
+                    <tbody>
+                        <tr>
+                            <th rowspan="1" colspan="1">Specification</th>
+                            <th rowspan="1" colspan="1">
+                                <abbr title="First Working Draft">FPWD</abbr>
+                            </th>
+                            <th rowspan="1" colspan="1">
+                                <abbr title="Candidate Recommendation">CR</abbr>
+                            </th>
+                            <th rowspan="1" colspan="1">
+                                <abbr title="Proposed Recommendation">PR</abbr>
+                            </th>
+                            <th rowspan="1" colspan="1">
+                                <abbr title="Recommendation">Rec</abbr>
+                            </th>
+                        </tr>
+                        <tr>
+                            <th rowspan="1" colspan="1">
+                                Solid Protocol
+                            </th>
+                            <td class="WD1" rowspan="1" colspan="3">
+                                Month YYYY
+                            </td>
+                        </tr>
+                    </tbody>
+                </table>
+
+            </div>
+        </section>
+
+
+        <section id="coordination">
+            <h2>Coordination</h2>
+            <p>
+                For all specifications, this Working Group will seek <a href="https://www.w3.org/Guide/Charter.html#horizontal-review">horizontal review</a>
+ for accessibility, internationalization, performance, privacy, and 
+security with the relevant Working and Interest Groups, and with the <a href="https://www.w3.org/2001/tag/" title="Technical Architecture Group">TAG</a>. Invitation for review must be issued during each major standards-track document transition, including <a href="https://www.w3.org/Consortium/Process//#RecsWD" title="First Public Working Draft">FPWD</a> and at least 3 months before <a href="https://www.w3.org/Consortium/Process/#RecsCR" title="Candidate Recommendation">CR</a>, and should be issued when major changes occur in a specification.
+            </p>
+
+            <p>
+                Additional technical coordination with the following Groups will be made, per the <a href="https://www.w3.org/Consortium/Process/#WGCharter">W3C Process Document</a>:
```

Comment:
```suggestion
                Additional technical coordination with the following Groups will be made, per the <a href="https://www.w3.org/Consortium/Process/#WGCharter">W3C Process Document</a>.
```

Reply:
Again, the line ending with the colon is introducing a list, albeit a list which the boilerplate split into two sections with `H3` elements (and which could probably lose the first H3 and the entire second section, since the second is commented out).

It might be clearer that this is the case (and that the colon is appropriate) if the H3 elements were converted into an outer UL enclosing the DLs.

Reply:
A section is _semantically_ different from a list item, independent of the syntactic boilerplate conversion into HTML. Sections are thematic divisions of the body of a text (or chapter), and only constitute a sequence because of that; i.e. they are lists on a meta-level. Lists in the proper sense, however, are on an object-level: they are already existing divisions, that the author wants to put into writing. Since the colon's semantics apply to the object-level, I would still argue that it is inappropriate to use it at the end of an introduction to sections.

Reply:
It's not an introduction to sections, even though the currently next HTML tag (which I'm about to suggest be changed) is an H3.  Regrettably, many boilerplates in use by W3C have not been reviewed with a strong focus on proper punctuation, even though the Style Guide (which itself is a relatively new creation) provides clear guidance.

Reply:
In other words, the _semantic_ meaning of HTML tags is not always appropriate for the _semantic_ meaning of the content being presented via that HTML. A great many people use the HTML that makes the most immediate sense for the rendered presentation they desire others to see in their browsers — regardless of the semantics implied by that HTML markup. We might hope that W3 documents were not constructed in such fashion, but that hope will always be forlorn, due to the (strongly desired and encourage!) breadth of the background experience of the Authors and Editors of those documents.

Reply:
Please provide a complete quotation of the CMOS "discouragement" to which you refer, as I think "discouragement" is a misunderstanding of what is written there.

Reply:
Different wording helps with the pain of using a full-stop instead of a colon, which I still believe to be the most appropriate punctuation here, even considering all other options.
```suggestion
Per the <a href="https://www.w3.org/Consortium/Process/#WGCharter">W3C Process
	Document</a>, additional technical coordination will be made with the Groups
	listed below.
```